src/field337.json

```diff
@@ -0,0 +1,12 @@
+[
```

Comment:
Move to field33XUtils and export as constant
`export const f337Variables = [..]`

Reply:
Fixed